src/containers/AllStream/index.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import Image from 'next/image';
+
+import CardSvg from '../../../public/images/card.svg';
+import HelpSvg from '../../../public/images/help.svg';
```

Comment:
Rename to `CardSvg` to `allStreamLogo` (change to file name too)

Rename `helpSvg` to `helpLogo` (do not set the first letter of an image uppercase)